docs/spec-conformance.md

```diff
@@ -15,3 +15,5 @@ v1.1.0-rc.1  | `.domainname`                            | [#3600](https://github
 v1.1.0-rc.1  | `.[]mounts.uidMappings`                  | [#3717](https://github.com/opencontainers/runc/pull/3717)
 v1.1.0-rc.1  | `.[]mounts.gidMappings`                  | [#3717](https://github.com/opencontainers/runc/pull/3717)
 v1.1.0-rc.1  | `SECCOMP_FILTER_FLAG_WAIT_KILLABLE_RECV` | TODO
+v1.1.0-rc.2  | time namespaces                          | TODO
+v1.1.0-rc.2  | rsvd hugetlb cgroup                      | TODO
```

Comment:
Should we create tracking tickets for these (if there's none yet)? and link to those (after TODO)

Reply:
- https://github.com/opencontainers/runc/issues/3859